shapeshift/web
# fix: txid not found when clicking the transaction toast immediately

## Description

Added 5 seconds setTimeout on toast to fix the txid  not found issue.

## Notice




## Pull Request Type


## Issue (if applicable)

After completing a successful send of a BTC transaction, a green transaction toast window will appear in the top right of your screen. Clicking the link immediately after the BTC send in the transaction toast will take you to an invalid explorer page because the TXID is not found yet.



## Risk

No risks, non-breaking.



## Testing

1. Make a BTC send transaction.
2. Observe the toast window pop up in the top right of the screen.
3. Immediately click on the link within the toast window to view transaction on an explorer page.



## Screenshots (if applicable)
